src/components/NetworkStatusService.js

```diff
@@ -60,13 +58,8 @@ goog.require('ga_waitcursor_service');
     var count = 0;
     var promise;
     this.$get = function($document, $rootScope, $timeout, $window,
-        gaBrowserSniffer, gaGlobalOptions) {
-      if (!gaBrowserSniffer.mobile) {
-        return {
-          offline: false,
-          check: function() {}
-        };
-      }
+        gaGlobalOptions) {
+
```

Comment:
On app launch: 
    If offline  -> launch checker requests every 2sec until it gets online again 
    If online  -> If an http request failed -> launch checker to see if really offline -> If offline -> launch checker every 2 sec ....